app/scripts/vueCore.js

```diff
@@ -139,7 +139,7 @@
               self_.checkForDropDownCharts();
               _.every(self_.groups, function(group) {
                 if (group.type === attrData.group_type) {
-                  if (group.attributes.length < 31) {
+                  if (group.attributes.length < 30) {
```

Comment:
dc filtering is not when there are more than one patient or sample groups if it 31. with 30 its working fine